OpenCorePkg.dsc

```diff
@@ -398,10 +398,8 @@
   #
   MdeModulePkg/Universal/Disk/RamDiskDxe/RamDiskDxe.inf
 
-  CryptoPkg/Library/IntrinsicLib/IntrinsicLib.inf
-
```

Comment:
Were these lines ever needed (also below in OpenDuetPkg)? I hope I'm not missing something, but AFAICT not.